<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbaqusExecutor</class>
 <widget class="QMainWindow" name="AbaqusExecutor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>658</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Abaqus Exector - Execute Your Abaqus Model Fluently</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/Qt.ico</normaloff>:/icons/icons/Qt.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>&amp;Setting</string>
    </property>
    <addaction name="actionEnvironment"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionModel"/>
    <addaction name="actionSubmit"/>
    <addaction name="actionOutput"/>
    <addaction name="actionDraw"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionCascade"/>
    <addaction name="actionTile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuWindows"/>
   <addaction name="menuSetting"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/new.png</normaloff>:/icons/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/saveas.png</normaloff>:/icons/icons/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbaqus">
   <property name="text">
    <string>Abaqus</string>
   </property>
  </action>
  <action name="actionAbaqusCAE">
   <property name="text">
    <string>Abaqus CAE</string>
   </property>
  </action>
  <action name="actionEnvironment">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/project-files.png</normaloff>:/icons/icons/project-files.png</iconset>
   </property>
   <property name="text">
    <string>Environment</string>
   </property>
  </action>
  <action name="actionSubmit">
   <property name="text">
    <string>Submit Job</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/cascade.png</normaloff>:/icons/icons/cascade.png</iconset>
   </property>
   <property name="text">
    <string>Window Cascade</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/tile.png</normaloff>:/icons/icons/tile.png</iconset>
   </property>
   <property name="text">
    <string>Window Tile</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="text">
    <string>Create Model</string>
   </property>
  </action>
  <action name="actionOutput">
   <property name="text">
    <string>Extract OutputData</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="text">
    <string>Draw Figures</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCascade</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>cascadeSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTile</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>tileSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>AbaqusExecutor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
